trackintel/io/postgis.py

```diff
@@ -73,16 +74,23 @@ def write_positionfixes_postgis(
     --------
     >>> df.as_positionfixes.to_postgis(conn_string, table_name)
     """
-
     # make a copy in order to avoid changing the geometry of the original array
     positionfixes_postgis = positionfixes.copy()
-
-    # If this GeoDataFrame already has an SRID, we use it, otherwise we default to WGS84.
     if positionfixes_postgis.crs is not None:
         srid = int(positionfixes_postgis.crs.to_epsg())
-    else:
-        srid = 4326
-    positionfixes_postgis["geom"] = positionfixes_postgis["geom"].apply(lambda x: WKTElement(x.wkt, srid=srid))
+        geom_schema = Geometry("POINT", srid)
```

Comment:
They should be tested with the second method `test_no_crs_setting`? But it seems that this is not recognised. I have to say, having the read and write in the same method is unelegant, maybe we should change that somehow.

Reply:
I guess this has to do with the failed tests. Since it failed somewhere in between the codecov will not recognize these lines as "covered".
As for the file structure, we can open an issue and look at how other repo deal with this situation